lib/tasks/evm_export_import.rake

```diff
@@ -68,6 +68,14 @@ namespace :evm do
 
       exit # exit so that parameters to the first rake task are not run as rake tasks
     end
+
+    desc 'Exports all custom buttons to a single YAML file'
+    task :custom_buttons => :environment do
```

Comment:
@d-m-u - I believe the task needs to be modified to accept arguments something like this:
```
task :custom_buttons, [:directory] => :environment do |t, args|
directory = args[:directory]
```